mne/io/cnt/cnt.py

```diff
@@ -129,34 +129,27 @@ def _get_cnt_info(input_fname, eog, ecg, emg, misc, data_format, date_format):
         session_date = read_str(fid, 10)
         time = read_str(fid, 12)
         date = session_date.split('/')
-        if len(date) != 3:
-            meas_date = -1
-        else:
+        if len(date) == 3 and len(time) == 3:
             if date[2].startswith('9'):
                 date[2] = '19' + date[2]
             elif len(date[2]) == 2:
                 date[2] = '20' + date[2]
             time = time.split(':')
-            if len(time) == 3:
-                if date_format == 'mm/dd/yy':
-                    pass
-                elif date_format == 'dd/mm/yy':
-                    date[0], date[1] = date[1], date[0]
-                else:
-                    raise ValueError("Only date formats 'mm/dd/yy' and "
-                                     "'dd/mm/yy' supported. "
-                                     "Got '%s'." % date_format)
-                # Assuming mm/dd/yy
-                date = datetime.datetime(int(date[2]), int(date[0]),
-                                         int(date[1]), int(time[0]),
-                                         int(time[1]), int(time[2]))
-                meas_date = calendar.timegm(date.utctimetuple())
-            else:
-                meas_date = -1
-        if meas_date < 0:
-            warn('  Could not parse meas date from the header. Setting to '
-                 '[0, 0]...')
-            meas_date = 0
+            if date_format == 'dd/mm/yy':
```

Comment:
Are these lines covered?

Reply:
I don't know, I just unindented these lines (during refactoring of `meas_date` conditional) so it should be in the same state as `master`

Reply:
Lines *138:152*  are not covered. See the output of ` py.test --cov=mne -r sx mne/io/cnt/cnt.py --cov-report term-missing`: 
```sh
mne/io/cnt/cnt.py                                             202    186     66      0     6%   93, 100-292, 359-364, 372-421
```
Actually most of the file is [not covered](https://codecov.io/gh/mne-tools/mne-python/pull/4542/src/mne/io/cnt/cnt.py). 

Should we address in this PR (or let it as it is)? cc: @agramfort

Reply:
no idea. @massich do you want to dig into the file format to cover these lines?

Reply:
I wont have the time right now, but I can do it in a separated PR and add it to my todo.